tests/test_lens.py

```diff
@@ -28,7 +29,9 @@
 
 from connect.governance import Governance
 
-import tempfile
+##################################################
```

Comment:
Can you add to documentation (probably just in this file or in a tests readme) instructions on what components tests need?
Looks like: init function, test function.
How do we know what model + data to pass to the init function?

Reply:
I think this part is still pending?

Reply:
@fabrizio-credo this sort of high-level description is a must.

It's not clear why we use the "credit" data for one test and the "classification" data for all the others. If someone wants to write a new test, they should be able to understand what exactly they need: which assume statements are the defaults to include, which data, under what circumstances does it make sense to modify conftest, etc.?

Reply:
Added a readme file. Please consider this a prototype. I will expand on it when I finish the remaining work on tests during the week.